src/Uno.UI/UI/Xaml/Controls/TextBlock/TextBlock.cs

```diff
@@ -695,7 +695,12 @@ private void UpdateInlines(string text)
 				// KNOWN ISSUE:
 				// On UWP the 'click' event is raised **after** the PointerReleased ... but deferring the event on the Dispatcher
 				// would move it after the PointerExited. So prefer to raise it before (actually like a Button).
-				that.FindHyperlinkAt(e.GetCurrentPoint(that).Position)?.ReleasePointerPressed(e.Pointer);
+				if (!(that.FindHyperlinkAt(e.GetCurrentPoint(that).Position)?.ReleasePointerPressed(e.Pointer) ?? false))
+				{
+					// We failed to find the hyperlink that made this capture but we ** silently ** removed the capture,
```

Comment:
Confirm my understanding, if the pointer is released outside an hyperlink, we won't "find" it, so we abort captures on all our hyperlinks?

Reply:
A pointer up will release the capture made for it. The issue is that we muted the event `PointerCaptureLost` (to match UWP behavior), but we didn't find the hyperlink which made the capture, so here we only make sure to notify the hyperlink that had captured this pointer to go back in default state.